addons/point_of_sale/models/account_move.py

```diff
@@ -9,6 +9,7 @@ class AccountMove(models.Model):
 
     pos_order_ids = fields.One2many('pos.order', 'account_move')
     pos_payment_ids = fields.One2many('pos.payment', 'account_move_id')
+    pos_refunded_invoice_ids = fields.Many2many('account.move', 'refunded_invoices', 'refund_account_move', 'original_account_move')
```

Comment:
Maybe this field could be interesting in "point_of_sale" module. We could use it for other loca.
But for now, we can let it here, it's just a note to think about it.